packages/esm-patient-chart-app/src/visit/visits-widget/visit-detail-overview.component.tsx

```diff
@@ -64,7 +64,7 @@ function VisitDetailOverviewComponent({ patientUuid }: VisitOverviewComponentPro
                         </div>
                       </div>
                       <div>
-                        <ExtensionSlot name="visit-detail-overview-actions" state={{ patientUuid, visit }} />
+                        <ExtensionSlot name="visit-detail-overview-actions-slot" state={{ patientUuid, visit }} />
```

Comment:
I actually prefer that we not have `-slot` on slot names. They're already namespaced, so it doesn't really add anything.

Reply:
removed